table_test.go

```diff
@@ -1419,3 +1420,97 @@ func Test_RowWriter(t *testing.T) {
 		}
 	}
 }
+
+// Test_Table_Size ensures the size of the table increases by the size of the inserted data.
+func Test_Table_Size(t *testing.T) {
+	test := func(isArrow bool) {
+		c, table := basicTable(t)
+		defer c.Close()
+
+		before := table.ActiveBlock().Size()
+
+		samples := dynparquet.Samples{{
```

Comment:
nit: why not just use `dynparquet.NewTestSamples` and modify those if you need to? It's less verbose

Reply:
simply forgot about it